roles/apache/handlers/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: restart apache
+  service: name=apache2 state=restarted
+
+- name: reload apache
+  service: name=apache2 state=reloaded
```

Comment:
What happens if we have a restart and a reload triggered? Does apache handle this well, sending one right after another? This is what got us in trouble with haproxy.